### PR TITLE
[MU4] Instruments panel simplification

### PR DESCRIPTION
**Pull request by RomanPudashkin**
- disabled an ability to work with doubling instruments
- fixed visual problems in instruments panel view
- implemented instrument change element
- collapse delegate when drag is active
- fixed moving staves to another parent
- refactored moving of staves
- fixed opening of popups when there is no much free space
- renamings in instruments dialog

### CODE REVIEW

**Comment by RobFog** [comment]
"disabled an ability" → "disabled the ability"
"no much free space" → "not much free space"
?

File: mu4/notation/internal/instrumentsconverter.cpp [hunk 1]
```diff
@@ -0,0 +1,127 @@
+#include "instrumentsconverter.h"
+
+#include "libmscore/instrument.h"
+
+using namespace mu::notation;
+using namespace mu::instruments;
+
+Ms::Instrument InstrumentsConveter::convertInstrument(const Instrument& instrument)
```
Comment (cbjeukendrup) [inline mu4/notation/internal/instrumentsconverter.cpp]: Typo: InstrumentsConveter → InstrumentsConverter

File: src/framework/uicomponents/qml/MuseScore/UiComponents/StyledPopup.qml [hunk 2]
```diff
@@ -83,7 +85,7 @@ Popup {
 
             Rectangle {
                 color: fillColor
-                border { width: 1; color: borderColor }
+                border { width: root.borderWidth; color: borderColor }
```
Comment (Eism) [inline src/framework/uicomponents/qml/MuseScore/UiComponents/StyledPopup.qml:88]: root.borderColor

File: mu4/notation/internal/instrumentsconverter.cpp [hunk 3]
```diff
@@ -0,0 +1,127 @@
+#include "instrumentsconverter.h"
```
Comment (Eism) [inline mu4/notation/internal/instrumentsconverter.cpp]: please, add license

File: src/framework/uicomponents/qml/MuseScore/UiComponents/StyledPopup.qml (hunk 2 above)
Comment (RomanPudashkin) [reply src/framework/uicomponents/qml/MuseScore/UiComponents/StyledPopup.qml]: fixed

File: mu4/notation/internal/instrumentsconverter.cpp (hunk 1 above)
Comment (RomanPudashkin) [reply mu4/notation/internal/instrumentsconverter.cpp]: fixed, thanks

**Review by Eism — APPROVED** [review]

**Pull request merged**